lib/src/mock_location.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:math';
+
+import 'constants.dart' show random;
+
+final R = 6372800;
+
+class Location {
+  double lat;
+  double lon;
+  Location(this.lat, this.lon);
+}
+
+/// Generate random latitude longitude in `radius` from `centerLat` to `centerLon`, inclusive.
+/// If any argument is null, random location will be returned.
+Location mockLocation([double centerLat, double centerLon, int radius]) {
+
+  if (centerLat == null || centerLon == null || radius == null) {
+    return Location(random.nextDouble() * 90, random.nextDouble() * 180);
```

Comment:
I see you used Decimal Degrees (DD). It states at https://en.wikipedia.org/wiki/Decimal_degrees that values can go from -90 to 90 for latitude and -180 to 180 for longitude. Is this right ?

Reply:
You right. I missed negative ranges.

Reply:
Yeah, I see. Try to fix this and other review comments so I can successfully merge and resolve this pull request :smile: :rocket: